src/krc_server.erl

```diff
@@ -243,7 +243,7 @@ handle_info({'EXIT', Pid, {shutdown, expired}},
             #s{client=Client, ip=IP, port=Port, pids=Pids} = S) ->
   ?hence(lists:member(Pid, list_pids(Pids))),
   ?debug("Krc EXIT ~p: expired connection", [Pid]),
-  telemetry_event([process, expired], #{pid => Pid}),
```

Comment:
This name change here from `process` -> `connection`. Does it make sense to do the same change on row 238? All the processes are connections right?